core/src/main/java/discord4j/core/object/ApplicationTeamMember.java

```diff
@@ -117,7 +117,7 @@ public Mono<User> getUser(EntityRetrievalStrategy retrievalStrategy) {
      * @return The user's role on the team.
      */
     public TeamMemberRole getRole() {
-        return TeamMemberRole.of(data.role());
+        return data.role().toOptional().map(TeamMemberRole::of).orElse(TeamMemberRole.ADMIN);
```

Comment:
in what cases this info not come?

Reply:
If the user is the owner of the team, he don't have any role

Reply:
ohhh that case...  yeah unless add a OWNER role but ADMIN works too then.